generators/cypress/templates/src/test/javascript/cypress/support/commands.ts.ejs

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/no-namespace */
+/* eslint-disable @typescript-eslint/no-use-before-define */
+// eslint-disable-next-line spaced-comment
+/// <reference types="cypress" />
+
+// ***********************************************
+// This commands.ts shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+
+// ***********************************************
+// Begin Specific Selector Attributes for Cypress
```

Comment:
Why placing this selectors in commands.ts ? I would have placed them in a selectors file